src/setfit/modeling.py

```diff
@@ -120,10 +124,11 @@ def forward(
 
         x = features["sentence_embedding"] if is_features_dict else features
         logits = self.linear(x)
-        if self.out_features == 1:  # only has one target
-            outputs = torch.sigmoid(logits)
-        else:  # multiple targets
-            temperature = temperature or self.temperature
+
+        temperature = temperature or self.temperature
+        if self.multitarget or self.out_features == 1:  # only has one target or multiple targets per item
```

Comment:
Should also update line:163 to use `BCELoss` when using multi-label classification